Comp2024/src/main/java/frc/robot/lib/util/CTREConfigs6.java

```diff
@@ -154,6 +154,10 @@ public static TalonFXConfiguration intakeRotaryFXConfig( )
   public static CANcoderConfiguration intakeRotaryCancoderConfig( )
   {
     CANcoderConfiguration config = new CANcoderConfiguration( );
+
```

Comment:
The easiest way to write a new config (CANcoder or TalonFX) is to copy the commented out lines from the example into your new config object. Then uncomment the ones needed. So there should be 3 lines below: one for AbsoluteSensorRange, SensorDirection, and MagnetOffset. You've got two.